lib/players/computer.rb

```diff
@@ -0,0 +1,122 @@
+require 'pry'
+
+module Players
```

Comment:
Hi Luke,

Can you please help me to see what you see here? As far as I can tell, this
class *does* have access to the game board and is able to get the
win_combinations from it.

   - In environment.rb
   <https://github.com/kkterai/ttt-with-ai-project-v-000/blob/master/config/environment.rb>,
   it says  "require_all 'lib'" so I don't understand it as an environment
   issue.
   - The board is also referenced throughout the class, including
   #win(board) which iterates through win_combinations.

Thanks in advance for your help in understanding this and we'll revise with
your response. :-) Best, Kenlyn

On Mon, Sep 12, 2016 at 12:39 PM, Luke Ghenco <notifications@github.com>
wrote:

> In lib/players/computer.rb
> <https://github.com/learn-co-students/ttt-with-ai-project-v-000/pull/290#discussion_r78439985>
> :
>
> > @@ -0,0 +1,122 @@
> > +require 'pry'
> > +
> > +module Players
>
> this class should have access to the game board and be able to get the
> win_combinations from that
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn-co-students/ttt-with-ai-project-v-000/pull/290/files/f94c72c0d4e70bec9233daa06f71e35810e1928f#r78439985>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AS4BkW0drU1EUIhEqiGnMJUAATCA4Qivks5qpaphgaJpZM4J66XR>
> .
>